server/@types/shared/models/PersonType.ts

```diff
@@ -0,0 +1,6 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+export type PersonType = 'FullPerson' | 'RestrictedPerson' | 'UnknownPerson' | 'FullPersonInfo' | 'RestrictedPersonInfo';
```

Comment:
I know this is out of scope for this PR but it feels a bit weird to me that 'FullPerson' and 'FullPersonInfo' belong in the same type...

Reply:
yeah I hadn't seen that... I'm not entirely sure I understand what the difference is between `FullPerson` and `FullPersonInfo` added here https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/804

Reply:
I think it might be left in from some changes that were made... `Person` was going to become `PersonInfo` but is now just `Person` (and same for other types?)